lib/modules.nix

```
@@ -554,11 +554,11 @@ let
               else
                 mapAttrs
                   (n: option:
-                    [{ inherit (module) _file; options = option; }]
+                    [{ inherit (module) _file; _loc = builtins.unsafeGetAttrPos n subtree; options = option; }]
```

```suggestion
                    [{ inherit (module) _file; pos = builtins.unsafeGetAttrPos n subtree; options = option; }]
```

`loc` would be confusing since it's often used for the option path in other parts of the code.
And no need to mark it with `_`, this code here is internal anyways :)